test/topology_custom/test_raft_no_quorum.py

```diff
@@ -54,7 +58,7 @@ async def test_cannot_add_new_node(manager: ManagerClient) -> None:
                          manager.server_stop_gracefully(servers[3].server_id))
 
     logger.info("starting a fifth node with no quorum")
-    await manager.server_add(expected_error="raft operation [read_barrier] timed out",
+    await manager.server_add(expected_error="raft operation [read_barrier] timed out, no group0 quorum",
```

Comment:
In the last commit you additionally match "no group0 quorum, total voters 2, alive voters 1". Maybe you could also match it here and in other tests?

Reply:
I wanted to check somewhere the logic which calculates alive voters. I did it once. The focus of other tests is on the specific scenarios where this "no group0 quorum" can happen, so I check just that. We could add the specific numbers of voters, but that would be redundant.